### DISCUSSION TITLE
ph_mat.x error

### DISCUSSION THREAD

**Discussion by callas7** [nguyen-group/QERaman#23] [Q&A]
Hello dear professor, 
I hope you are doing well. I run the scf.in and the bands.in but when it comes to ph.in I got an error, the code starts and before it finished calculations it stopped. I will leave the output file and the error given by the hpc. 
Best regards.
[hpc_Error.docx](https://github.com/user-attachments/files/17541693/hpc_Error.docx)
[ph.out.docx](https://github.com/user-attachments/files/17541694/ph.out.docx)

**Comment by nguyen-group** [comment]
Hi,

This error "forrtl: No such file or directory" is related to HPC's library environment. You need to link to the correct library for MPI, etc. You can try with ph.x (note that you need to comment electron_phonon = 'epc') to see what happens. If same error, the problem might not in ph_mat.x. If there are no errors, you can try to recompiler the ph_mat.x.

Nguyen
